bin/main.rb

```diff
@@ -1,3 +1,116 @@
 #!/usr/bin/env ruby
+# rubocop:disable Metrics/BlockNesting
+require_relative 'board'
```

Comment:
You are requiring a file that doesn't exist, and it is causing errors in running the codes.